Resources/doc/config.rst

```diff
@@ -306,15 +306,13 @@ It is possible to connect to several mongodb servers on one connection if
 you are using a replica set by listing all of the servers within the connection
 string as a comma separated list.
 
-.. configuration-block::
```

Comment:
I think I'd rather keep them and add other configurations instead (in another PR)